PhysicsTools/Heppy/python/physicsobjects/Tau.py

```diff
@@ -46,27 +48,68 @@ def dxy_approx(self, vertex=None):
         return ( - (vtx.x()-vertex.position().x()) *  p4.y()
                  + (vtx.y()-vertex.position().y()) *  p4.x() ) /  p4.pt()
 
+    def countWP(self,name, allWP=False):
+        '''Returns the number of Working Points 
+        that are passed for given isolation name. 
```

Comment:
too vague, the user cannot understand what is the returned value from this docstring.
should give the list of working points and the returned value (maybe with an example)
should explain what happens when the wp state is read from mini aod
should explain what name and allWP should be and what they do